packages/@storefront/flux-capacitor/test/unit/core/actions/creators.ts

```diff
@@ -1163,6 +1185,29 @@ suite('ActionCreators', ({ expect, spy, stub }) => {
       });
     });
 
+    describe('selectMultiplePastPurchaseRefinements()', () => {
+      const navigationId = 'colour';
+
+      it('should return a batch action with RESET_PAGE and one SELECT_REFINEMENT for each index given', () => {
+        const indices = [1, 2, 3];
+        const resetPastPurchasePageAction: any = { a: 'b' };
+        const selectPastPurchaseRefinementAction: any = { c: 'd' };
+        const selectPastPurchaseRefinement = stub(ActionCreators, 'selectPastPurchaseRefinement')
```

Comment:
Same as above re: inaccurate return value. Does this accurate test the new functionality?